migration_steps/transform_casrec/transform/transform_tests/transform_data_tests/apply_conditions/cases_capitalise_first_letter.py

```diff
@@ -0,0 +1,23 @@
+from pytest_cases import case
+import pandas as pd
+
+
+case(id="bring back capitalised strings")
+
+
+def case_less_than():
+    test_data = {
+        "c_test_col": ["BOB", "mary-Jane", "hecTor Hamster", ""],
+    }
+    test_data_df = pd.DataFrame(test_data, columns=[x for x in test_data])
+
+    conditions = {"capitalise_first_letter": {"cols": ["c_test_col"]}}
+
+    expected_data = {
+        "c_test_col": ["Bob", "Mary-jane", "Hector hamster", ""],
```

Comment:
I think you should have "Hector Hamster" here, as the example in the story
![image](https://user-images.githubusercontent.com/146390/141956213-78dd54d8-bd94-4402-821b-ee6df6ae8826.png)